### DISCUSSION TITLE
코멘트 수정 URI 변경 제안 드립니다.

### DISCUSSION THREAD

**Discussion by sanaandmomo** [woowacourse-teams/2022-ternoko#262] [Polls]
## As is
`/api/interviews/{interviewId}/comments/{commentId}`

## To be
`/api/interviews/comments/{commentId}`

`commentId`가 `pk`라서 `interviewId`가 굳이 필요할까용??

**Comment by her0807** [comment]
아래와 같은 이유로 ToBe 추천합니다. 


1. 안쓰는 리소스를 굳이 명시할 필요가 없다,
2. 인터뷰 안의 코멘트가 업데이트 된다고 해서 인터뷰의 변경 사항이 전파되지 않는다

**Reply by HyeonbinSa** [reply]
동의합니다.

**Comment by dongho108** [comment]
interviews만 넣을바에 `/api/comments/{commentId}`가 나을 것 같아요
